website/docs/d/scc_posture_collector.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_collector"
+description: |-
+  Get information about collector
+subcategory: "Posture Management"
```

Comment:
I think the subcategory should be "Security and Compliance Center" in all docs

Reply:
@hkantare , modified to "Security and Compliance Center" in all docs